### PR TITLE
stream server: Bind to all addresses

### PR DESCRIPTION
**Pull request by mrunalp**
If stream server address isn't specified, then we
should bind to all addresses.

Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by giuseppe** [comment]
/retest

LGTM

**Comment by haircommander** [comment]
rebase to fix test failures

File: server/server.go [hunk 1]
```diff
@@ -415,10 +414,13 @@ func New(
 	s.restore(ctx)
 	s.cleanupSandboxesOnShutdown(ctx)
 
+	var bindAddressStr string
 	bindAddress := net.ParseIP(config.StreamAddress)
 	if bindAddress == nil {
-		hostIPs := s.getHostIPs(config.HostIP)
-		bindAddress = hostIPs[0]
+		// If bind address isn't valid or unset, then listen on all IP addresses
+		bindAddressStr = ""
```
Comment (haircommander) [inline server/server.go]: do we need this case? can't we just check if bindAddress != nil, and if so set bindAddressStr? `var bindAddressStr string` sets it as ""

Comment (mrunalp) [reply server/server.go]: oops, yeah. we can do that in a follow cleanup PR.

Comment (haircommander) [reply server/server.go]: fixing it here https://github.com/cri-o/cri-o/pull/3153

**Pull request merged**